docs/en/latest/plugins/limit-conn.md

```diff
@@ -40,7 +40,7 @@ Limiting request concurrency plugin.
 | burst              | integer | required    |         | burst >= 0                                                                                | the number of excessive concurrent requests (or connections) allowed to be delayed.                                                                                                                                                                                                                                                                                                                                                                                                                       |
 | default_conn_delay | number  | required    |         | default_conn_delay > 0                                                                    | the latency seconds of request when concurrent requests exceeding `conn` but below (`conn` + `burst`).                                                                                                                                                                                                                                                                                                                                                                                                                                   |
-| delay_strict_mode  | boolean | optional    | false   | [true,false]                                                                              | enable the strict mode of the latency seconds. If you set this option to `true`, it will run strictly according to the latency seconds you set without additional calculation logic.                                                                                                                                                                                                                                                                                                                               |               
+| delay_strict_mode  | boolean | optional    | false   | [true,false]                                                                              | enable the strict mode of the latency seconds. If you set this option to `true`, it will run strictly according to the latency seconds you set without additional calculation logic.                                                                                                                                                                                                                                                                                                                      |
```

Comment:
I think the option name is not so straightforward but I didn't come out with a more proper one :(

Reply:
😂

Reply:
What about `only_use_default_delay`?

Reply:
`only_use_default_delay`,   `use_default_conn_delay`,  `apply_default_conn_delay`,  `stint_default_conn_delay`, Which of the four do you think is better, or do you have better suggestions?

Reply:
I vote for the first one. What about others' opinions?

Reply:
@tokers  what do you think?

Reply:
Vote for the first one too, it's easy to understand.

Reply:
Okay, I will change it to this one. 
What about the test case I mentioned above? Any suggestions? @spacewander @tokers

Reply:
You can take a look at the test in https://github.com/apache/apisix/pull/4585